content/blog/setting-bios-and-storage-controller-properties-with-redfish.md

```diff
@@ -15,88 +16,40 @@ tags:
 ---
 <style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
 
-Updated March 5, 2024
-
-The concept of deferred / pending settings in the <a href="https://servermanagementportal.ext.hpe.com/docs/concepts/biosdatamodel/#bios-current-and-pending-areas" target="_blank">Bios</a> and `SmartStorageConfig` subsystems of HPE iLO 5 is briefly presented in the <a href="https://servermanagementportal.ext.hpe.com/" target="_blank">HPE Reference API</a> documentation. With two examples, this document illustrates what is happening when properties are modified in those subsystems using the Redfish REST API against HPE servers.
```

Comment:
Hi, Francois. I can't actually see a preview of this blog post, so bear with me as I imagine how it actually looks like on the screen ;-)

A quick question at the end of this sentence. Should it say "REST API against HPE servers." or "REST API with HPE servers."?